docs/Contributing/API-for-contributors.md

```diff
@@ -161,7 +161,7 @@ Returns the specs for all packs in the Fleet instance.
           "interval": 456,
           "snapshot": false,
           "removed": true,
-          "platform": "windows",
+          "platform": "Windows",
```

Comment:
```suggestion
          "platform": "windows",
```

I think the API requires a lowercase `windows` for the `platform` field. Can you please remove this change?

In most cases the capitalization used in the example JSON (example code) will be correct. This is because, the contributor that added these docs likely copied & pasted the request or response.

That being said, this might not be the case for fields like `description` and `name`. You caught a couple of places where we'd like to use uppercase for these fields 🎸